### ISSUE TITLE
Need to know why added two times model fit

### ISSUE THREAD

**Issue by swagat123** [deepakgupta1/Rakuten-Food-Image-Classification#1]
Hi 
Thank you for the code sharing. I have little doubts in the code , you have mentioned model.fit_generator two times . 1st time it ran with 2 epochs and 2nd time it ran with 200 epochs.
Before 2nd time model.fit_generator it compiled with learning rate 0.0001 where as 1st time it is 0.001.

My doubt is why this two time model.fit_generator.

I appreciate your help.

Regards
Swagat

**Comment by deepakgupta1** [comment]
Hey there. You can also notice that the first time I have frozen all but extra added layers and the second time I have trained all the layers. The idea is to first train only the extra added FC layers with a higher learning rate and then fine-tune the whole network with a smaller learning rate so the pre-trained weights (imagenet here) don't get too disturbed. I have mentioned this in my approach section also in README. Hope this helps. Cheers!

**Comment by deepakgupta1** [comment]
Closing, as I guess you have got your answer.

**Comment by swagat123** [comment]
Thank you for the explanation
